src/main/java/org/broadinstitute/hellbender/tools/walkers/UnmarkDuplicates.java

```diff
@@ -17,15 +17,44 @@
 import java.util.Collections;
 import java.util.List;
 
-@DocumentedFeature
+/**
+ * Removes duplicate read tags.
+ *
+ * <p> This tool "unmarks" duplicate reads in a SAM/BAM/CRAM file. Duplicate reads are defined as originated from a
+ * single fragment of DNA and can arise during sample preparation. </p>
+ *
+ *
+ * <h3>Input</h3>
+ * <p>
+ * A SAM/BAM/CRAM file that has been marked for duplicates.
+ * </p>
+ * <h3>Output</h3>
+ * <p>
+ * A SAM/BAM/CRAM file without isDuplicate on reads.
+ * </p>
+ *
+ * <h3>Usage example: </h3>
+ * <pre>
+ *     gatk UnmarkDuplicates \
+ *     -I marked_duplicates.bam \
```

Comment:
Since you will make modifications already, here's a minor thing to change. Can you de-indent the first line of the command so the command looks like below? Also, let's clean up the file naming to make it appear simpler.
```
gatk UnmarkDuplicates \
    -I clean.bam \
    -O unmarked.bam
```